shared/types/admin/DonationCertificate.ts

```diff
@@ -0,0 +1,8 @@
+export const DONATION_CERTIFICATE_FIRESTORE_PATH = 'donation-certificate';
+
+export type DonationCertificate = {
+	created: Date;
+	url: string;
+	country: string;
+	year: string;
```

Comment:
nit: number

Reply:
I actually used string on purpose because I use it only as a string